docs/error_codes.md

```diff
@@ -29,3 +29,4 @@ Only the positive error codes are listed below, although the custom device is al
 | `732150` | `732174` | [Scan Engine and EtherCAT](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device) |
 | `732200` | `732224` | [VeriStand Custom Device Development Tools](https://github.com/ni/niveristand-custom-device-development-tools) |
 | `732250` | `732274` | [Ballard ARINC-429](https://github.com/ni/niveristand-ballard-arinc429-custom-device) |
+| `732275` | `732299` | [Encoding and Decoding Library](https://github.com/ni/niveristand-ballard-arinc429-custom-device/tree/main/Source/Encoding%20and%20Decoding) |
```

Comment:
Can you start this at `732300`? We are leaving a gap between allocated ranges for now, to allow for an existing range to expand contiguously if needed.